README.md

```diff
@@ -129,20 +129,18 @@ To manually upgrade your local project:
 1. Run `npm update` to update node modules.
 1. Run `nox --session setup` to upgrade python packages.
 
-# Troubleshooting
+## Troubleshooting
 
-## Changing path or name of `server.py` something else.
+### Changing path or name of `server.py` something else
 
 If you want to change the name of `server.py` to something else, you can. Be sure to update `constants.ts` and `src\test\python_tests\lsp_test_client\session.py`.
 
 Also make sure that the inserted paths in `server.py` are pointing to the right folders to pick up the dependent packages.
 
-## Module not found errors.
+### Module not found errors
 
 This can occurs if `bundled/libs` is empty. That is the folder where we put your tool and other dependencies. Be sure to follow the build steps need for creating and bundling the required libs.
 
-Common one is `pygls` module not found.
+Common one is [_pygls_][pygls] module not found.
 
-# TODO: The maintainer of this repo has not yet edited this file
-
-**Repo Owner** Make sure you update this. As a repository owner you will need to update this file with specific instructions for your extension.
```

Comment:
This was intentional for the users of the template to edit this file before publishing. This readme is not relevant when it is consumed to build an extension.